std/traits.d

```diff
@@ -6839,3 +6839,348 @@ unittest
     struct BoolWrapper { bool value; }
     static assert(!ifTestable!(bool, a => BoolWrapper(a)));
 }
+
+/**
+ * Detect whether $(D X) is a type or not.
+ *
+ * Returns:
+ *     `bool`
```

Comment:
Sorry, I've missed this new DDOC feature. BTW you mix the old and the new syntax, despite the recommandation you've get in #3800.